.github/workflows/run-docker.yml

```diff
@@ -10,8 +10,29 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Build Docker image
-        run: docker build -f docker/jupyter.Dockerfile -t traffic/jupyter docker
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Import the library
-        run: docker run traffic/jupyter python -c "import traffic; print(traffic.__version__)"
+      # jupyter Docker image for users
+      - name: Build jupyter Docker image
+        run: docker build --file docker/jupyter.Dockerfile --tag ghcr.io/xoolive/traffic/jupyter:latest docker
+
+      - name: Import the library for testing
+        run: docker run ghcr.io/xoolive/traffic/jupyter:latest python -c "import traffic; print(traffic.__version__)"
+        
+      - name: Push image
+        run: docker push ghcr.io/xoolive/traffic/jupyter:latest
+
+      # dev Docker image
+      - name: Build poetry Docker image
+        run: docker build --file docker/poetry.Dockerfile --tag ghcr.io/xoolive/traffic/dev:latest .
+
+      # - name: Import the library for testing
```

Comment:
Same comment as above for the poetry version

Reply:
same comment as above as well :-)